pkg/testcase/dualstack.go

```diff
@@ -0,0 +1,104 @@
+package testcase
+
+import (
+	"github.com/rancher/distros-test-framework/shared"
+	"github.com/rancher/distros-test-framework/pkg/assert"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestDualStackIngress(deleteWorkload bool) {
+	_, err := shared.ManageWorkload("apply", "dualstack-ingress.yaml")
+	Expect(err).NotTo(HaveOccurred())
+	
+	testdata := map[string]string{
```

Comment:
since u wanted to have somehitng more verbose and kind of data driven
in go is a best practice create table driven tests

someething like : 

```
type testData struct {
	TestDataMap      anyname
	ExpectedIPFamily []string
}

type anyname struct {
	namespace string
	label     string
	svc       string
	exp       string
}


and then use it across your functions. 
its more organized 

This is only a advise, not a must do.
```

Reply:
event its just here , since u are using a lot of maps its a way to organize better and have more readability .
its a improvement, no need if u dont want to.